src/components/Quotes.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import './Quotes.css';
+
+const Quote = () => {
+  //   const [quote, setQuote] = useState('');
+  //   const [author, setAuthor] = useState('');
```

Comment:
Please kindly remove all unused code for it makes your codebase look untidy please lets stick the microverse clean code policy. it would be nice if you got rid of this because its not in use